Automattic-Tracks-iOS/Crash Logging/CrashLogging.swift

```diff
@@ -2,178 +2,251 @@ import Foundation
 import Sentry
 import CocoaLumberjack
 
+struct CrashLoggingInternals {
+    static var crashLogging: CrashLogging?
+}
+
 /// A class that provides support for logging crashes. Not compatible with Objective-C.
 public class CrashLogging {
 
-    /// A singleton is maintained, but the host application needn't be aware of its existence.
```

Comment:
No more static instance internally – makes things safer, easier to test, and lets us remove a bunch of code.